scanner/src/main/kotlin/Scanner.kt

```diff
@@ -86,7 +108,20 @@ fun scanOrtResult(
         }
 
     val scanResults = runBlocking {
-        scanner.scanPackages(packagesToScan, outputDirectory).mapKeys { it.key.id }
+        // Scan the projects from the ORT result.
+        val deferredProjectScan = async {
+            projectScanner.scanPackages(filteredProjectPackages, outputDirectory).mapKeys { it.key.id }
```

Comment:
I think we need to consider some exception catching here: maybe it would be worthwhile to catch exceptions thrown by one `scanPackages`, to be able to collect the results of the other ?

I imagine the case where someone run Scancode + FossID on a big project and e.g. FossID has an exception, it would be cool to nevertheless have the Scancode results.

Reply:
I agree that would be cool. However, I'll leave that as a future exercise, as [exception handling with Kotlin coroutines is hard](https://www.lukaslechner.com/why-exception-handling-with-kotlin-coroutines-is-so-hard-and-how-to-successfully-master-it/).